holidays/countries/cape_verde.py

```diff
@@ -0,0 +1,95 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.constants import OPTIONAL, PUBLIC
+from holidays.groups import ChristianHolidays, InternationalHolidays
+from holidays.holiday_base import HolidayBase
+
+
+class CapeVerde(HolidayBase, ChristianHolidays, InternationalHolidays):
+    """Cape Verde holidays.
+
+    References:
+        * [Public holidays in Cape Verde](https://en.wikipedia.org/wiki/Public_holidays_in_Cape_Verde)
+        * [Today's and Upcoming Holidays in Cape Verde](https://www.timeanddate.com/holidays/cape-verde/)
+        * [Democracy Day](https://www.officeholidays.com/holidays/cape-verde/cape-verde-democracy-day)
+        * [National Heroes Day](https://www.officeholidays.com/holidays/cape-verde/guinea-bissau-national-heroes-day)
+    """
+
+    country = "CV"
+    default_language = "pt_CV"
+    start_year = 1976
```

Comment:
If you plan to add subvision supports as well as listed in the new sources, see [italy.py](https://github.com/vacanza/holidays/blob/b870ccfbb9516fceb81d8f5a10f5a516cce5fc67/holidays/countries/italy.py#L27) for example.

List of Cape Verde's subdivision ISO codes: https://www.iso.org/obp/ui/#iso:code:3166:CV